openshift/template_insights-inventory.yaml

```diff
@@ -182,10 +189,10 @@ parameters:
   - description: 'The inventory service path prefix. Used to build the API url (/<INVENTORY_PATH_PREFIX>/<INVENTORY_APP_NAME>/api/v1/)'
     displayName: Inventory Path Prefix
     name: INVENTORY_PATH_PREFIX
-    value: /r/insights/platform
+    value: r/insights/platform
     required: false
   - description: 'The inventory service app name. Used to build the API url (/<INVENTORY_PATH_PREFIX>/<INVENTORY_APP_NAME>/api/v1/)'
     displayName: Inventory App Name
     name: INVENTORY_APP_NAME
-    value: inventory
+    value: inventory/api
```

Comment:
Also, I shouldn't have to name the app 'inventory/api' just to make things work.